components/admin_console/team_channel_settings/channel/details/channel_modes.tsx

```diff
@@ -11,7 +11,14 @@ import AdminPanel from 'components/widgets/admin_console/admin_panel';
 import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx';
 import LineSwitch from '../../line_switch';
 
-const SyncGroupsToggle = ({isSynced, isPublic, isDefault, onToggle}) => (
+interface Props {
```

Comment:
how would you suggest doing this one.  The 3 functional components have the same Prop

Reply:
I'm alright with using the same interface for all 3 components, we may as well stay DRY for the time being until the components need to change (if they do).